src/states/infiniteweightpeps.jl

```diff
@@ -49,6 +49,17 @@ function compare_weights(wts1::SUWeight, wts2::SUWeight)
     return sum(_singular_value_distance, zip(wts1.data, wts2.data)) / length(wts1)
 end
 
+function Base.show(io::IO, wts::SUWeight)
```

Comment:
```suggestion
function Base.show(io::IO, ::MIME"text/plain", wts::SUWeight)
```

I think the usual convention is to have the 2-argument `show` be something that is Julia-parsable, while the 3-argument is human-readable.
This is definitely a convention that is not always respected, and honestly the entire `show` ecosystem in Julia could do with some improvements, but here I think we can just follow that.

To see the difference, note that by default, displaying something calls the 3-arg show, which falls back to the 2-arg one. However, for example in `Vector` contexts you probably don't want to have this multiline printing, so you can compare these two cases.

Reply:
Currently I'm content with defining the human-readable `display` only.